crates/uv/tests/it/common/mod.rs

```diff
@@ -592,6 +592,59 @@ impl TestContext {
             filters.push((r"exit code: ".to_string(), "exit status: ".to_string()));
         }
 
+        for (version, executable) in &python_versions {
+            // Add filtering for the interpreter path
+            filters.extend(
+                Self::path_patterns(executable)
+                    .into_iter()
+                    .map(|pattern| (pattern.to_string(), format!("[PYTHON-{version}]"))),
+            );
+
+            // Add filtering for the bin directory of the base interpreter path
+            let bin_dir = if cfg!(windows) {
+                // On Windows, the Python executable is in the root, not the bin directory
+                executable
+                    .canonicalize()
+                    .unwrap()
+                    .parent()
+                    .unwrap()
+                    .join("Scripts")
+            } else {
+                executable
+                    .canonicalize()
+                    .unwrap()
+                    .parent()
+                    .unwrap()
+                    .to_path_buf()
+            };
+            filters.extend(
+                Self::path_patterns(bin_dir)
+                    .into_iter()
+                    .map(|pattern| (pattern.to_string(), format!("[PYTHON-BIN-{version}]"))),
+            );
```

Comment:
This is new, the rest is just moved up to ensure it comes first in the filter order.